src/given.jl

```diff
@@ -0,0 +1,39 @@
+"""
+    `@given d x`
```

Comment:
Regarding the name, I think that brevity is essential as this will be typed pretty often. I like  `@in` to denote that you are working within a given DataFrame, but `@use` also makes sense.